content/integrations/awslambda.md

```diff
@@ -21,27 +21,93 @@ If you haven't already, set up the [Amazon Web Services integration first](/inte
 In the Amazon Web Services integration tile, ensure that Lambda is checked under metric collection.
 
 To send custom metrics to Datadog, you must print a log line from your Lambda, using the following format:
-`MONITORING|unix_epoch_timestamp|value|metric_type|my.metric.name|#tag1:value,tag2`
 
-Please ensure the `unix_epoch_timestamp` is in seconds (not milliseconds).
+~~~
+MONITORING|unix_epoch_timestamp|value|metric_type|metric.name|#tag1:value,tag2
+~~~
+
+## Notes on each section
```

Comment:
H2 and H3 headers for this section are a bit heavy. We can just turn this whole thing into one list, like the [DogStatsD guide does](http://docs.datadoghq.com/guides/dogstatsd/#metricformat), i.e.

```
MONITORING|unix_epoch_timestamp|value|metric_type|metric.name|#tag1:value,tag2
```

- `MONITORING` – The Datadog Lambda integration watches Cloudwatch logs and pulls in any log entries that begin with this string.
- `unix_epoch_timestamp` – the log timestamp in seconds (NOT milliseconds)

etc